### PR TITLE
[VL][CI] Use pre-installed celeborn to avoid download failure

### PR DESCRIPTION
**Pull request by PHILO-HE**
Celeborn 0.3.2 & 0.4.0 have been pre-installed in CI docker.

### CODE REVIEW

**Comment by kerwin-zk** [comment]
LGTM. Thanks!

**Review by kerwin-zk — APPROVED** [review]

File: .github/workflows/velox_be.yml [hunk 1]
```diff
@@ -453,8 +453,8 @@ jobs:
       - name: TPC-H SF1.0 && TPC-DS SF10.0 Parquet local spark3.2 with Celeborn 0.4.0
         run: |
           $PATH_TO_GLUTEN_TE/$OS_IMAGE_NAME/gha/gha-checkout/exec.sh \
-          'wget https://archive.apache.org/dist/incubator/celeborn/celeborn-0.4.0-incubating/apache-celeborn-0.4.0-incubating-bin.tgz && \
-          tar xzf apache-celeborn-0.4.0-incubating-bin.tgz -C /opt/ && mv /opt/apache-celeborn-0.4.0-incubating-bin /opt/celeborn && cd /opt/celeborn && \
+          'cd /opt && mkdir -p celeborn-0.4.0 && \
```
Comment (kerwin-zk) [inline .github/workflows/velox_be.yml:456]: It is recommended that this folder be unified into celeborn.

File: .github/workflows/velox_be.yml [hunk 2]
```diff
@@ -469,8 +469,8 @@ jobs:
       - name: TPC-H SF1.0 && TPC-DS SF10.0 Parquet local spark3.2 with Celeborn 0.3.2
         run: |
           $PATH_TO_GLUTEN_TE/$OS_IMAGE_NAME/gha/gha-checkout/exec.sh \
-          'wget https://archive.apache.org/dist/incubator/celeborn/celeborn-0.3.2-incubating/apache-celeborn-0.3.2-incubating-bin.tgz && \
-          tar xzf apache-celeborn-0.3.2-incubating-bin.tgz -C /opt/ && mv /opt/apache-celeborn-0.3.2-incubating-bin /opt/celeborn && cd /opt/celeborn && \
+          'cd /opt && mkdir -p celeborn-0.3.2 && \
```
Comment (kerwin-zk) [inline .github/workflows/velox_be.yml:472]: ditto.

File: .github/workflows/velox_be.yml (hunk 1 above)
Comment (PHILO-HE) [reply .github/workflows/velox_be.yml]: Yes, just fixed. Thanks!

**Pull request merged**